packages/utils/src/lib/reports/utils.ts

```diff
@@ -324,3 +253,52 @@ export function compareIssues(a: Issue, b: Issue): number {
 
   return 0;
 }
+
+// @TODO rethink implementation
+export function applyScoreColor(
+  { score, text }: { score: number; text?: string },
+  style: Ansis = ansis,
+) {
+  const formattedScore = text ?? formatReportScore(score);
+
+  if (score >= SCORE_COLOR_RANGE.GREEN_MIN) {
+    return text
+      ? style.green(formattedScore)
+      : style.bold(style.green(formattedScore));
+  }
+
+  if (score >= SCORE_COLOR_RANGE.YELLOW_MIN) {
+    return text
+      ? style.yellow(formattedScore)
+      : style.bold(style.yellow(formattedScore));
+  }
+
+  return text
+    ? style.red(formattedScore)
+    : style.bold(style.red(formattedScore));
+}
+
+export function targetScoreIcon(
+  score: number,
+  targetScore?: number,
+  options: {
+    passIcon?: string;
+    failIcon?: string;
+    prefix?: string;
+    postfix?: string;
+  } = {},
+): string {
+  if (targetScore != null) {
+    const {
+      passIcon = '✅',
+      failIcon = '❌',
+      prefix = '',
+      postfix = '',
+    } = options; // ✅✓ ❌✗
```

Comment:
Not sure what to make of this comment :smile:

Reply:
removed